lib/src/logic.dart

```diff
@@ -423,9 +423,15 @@ class Logic {
     }
   }
 
-  /// Accesses the [index]th bit of this signal.
-  Logic operator [](int index) {
-    return slice(index, index);
+  /// Accesses the [index]th bit of this signal, where index can be [int] or [Logic].
+  /// Throws Exception when index is not in range or index is neither [int] or [Logic].
```

Comment:
Add a new-line between the summary and details of a documentation comment.  See: https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph